CMakeLists.txt

```diff
@@ -0,0 +1,115 @@
+###############################################################################
+#
+# (c) Copyright IBM Corp. 2016
+#
+#  This program and the accompanying materials are made available
+#  under the terms of the Eclipse Public License v1.0 and
+#  Apache License v2.0 which accompanies this distribution.
+#
+#      The Eclipse Public License is available at
+#      http://www.eclipse.org/legal/epl-v10.html
+#
+#      The Apache License v2.0 is available at
+#      http://www.opensource.org/licenses/apache2.0.php
+#
+# Contributors:
+#    Multiple authors (IBM Corp.) - initial implementation and documentation
+###############################################################################
+
+cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
+
+set(OMR_VERSION_MAJOR 0)
```

Comment:
Also, now that I think about it, this is really more of a place holder. Ideally we need figure out a system where we can store version information once and consume in both build systems